rules/exec/remote_commands/code_eval.yara

```diff
@@ -59,8 +59,24 @@ rule js_eval_near_enough_fromChar: high {
     $exec    = /[\s\{]eval\(/
     $decrypt = "String.fromCharCode"
 
+    $not_code_server = "fromCharCode(...codes: number[]): string;"
+    $not_grafana     = "self.webpackChunkgrafana=self.webpackChunkgrafana||[]).push("
+    $not_jupyter     = "self[\" webpackChunk_jupyterlab_application_top \"]=self[\" webpackChunk_jupyterlab_application_top \"]||[]).push("
+    $not_jupyter2    = "self[\" webpackChunk_JUPYTERLAB_CORE_OUTPUT \"] = self[\" webpackChunk_JUPYTERLAB_CORE_OUTPUT \"] || []).push([[132,7061]"
+    $not_jupyter3    = "self[\" webpackChunk_jupyterlab_application_top \"]=self[\" webpackChunk_jupyterlab_application_top \"]||[]).push([[227],{69119:"
+    $not_jupyter4    = "self[\" webpackChunk_jupyterlab_application_top \"]=self[\" webpackChunk_jupyterlab_application_top \"]||[]).push([[9296],{49296:"
+    $not_jupyter5    = "self[\" webpackChunk_jupyterlab_application_top \"]=self[\" webpackChunk_jupyterlab_application_top \"]||[]).push([[4470],{27902:"
```

Comment:
Since this is probably triggering on webpack, can we just use `webpackChunk` as a signal instead of these other jupyter ones?

Reply:
On second thought, with this many falses, I'd downgrade this rule to a medium.

Reply:
Yea, dropping the severities to medium and high respectively would make a huge difference.

Reply:
Updated in [`edf3d67` (#949)](https://github.com/chainguard-dev/malcontent/pull/949/commits/edf3d6766da6907f84f71a873df740019f04a81d).